### ISSUE TITLE
False "Multiple exports of name" error when two "export * as ..." contain same name. [import/export]

### ISSUE THREAD

**Issue by Stijnkool** [import-js/eslint-plugin-import#2290]
This seems related to https://github.com/import-js/eslint-plugin-import/issues/1834.
In that scenario there is no longer a problem. It's fixed.
I am having the same problem but with an extra level of indirection.
TypeScript does not seem to have a problem with the extra added level.
So I guess this plugin should allow it?

## Scenario in https://github.com/import-js/eslint-plugin-import/issues/1834
./A/A1.ts
```ts
export type X = "A1";
```
./A/A2.ts
```ts
export type X = "A2";
```
./A/index.ts
```ts
export * as A1 from "./A1";
export * as A2 from "./A2";
```
In  https://github.com/import-js/eslint-plugin-import/issues/1834 this would give an error `Multiple exports of name 'X'`.

**Comment by ljharb** [comment]
Duplicate of #2289.

**Comment by Stijnkool** [comment]
Oops, yea sorry
